Tests/js/views/services/assets/ez-serversideviewservice-tests.js

```diff
@@ -335,8 +335,138 @@ YUI.add('ez-serversideviewservice-tests', function (Y) {
         },
     });
 
+    notificationTest = new Y.Test.Case({
+        name: "eZ Server Side View Service notification test",
+
+        setUp: function () {
+            this.baseUri = '/Tests/js/views/services/';
+            this.app = new Y.Base();
+
+            Mock.expect(this.app, {
+                method: 'get',
+                args: ['baseUri'],
+                returns: this.baseUri,
+            });
+        },
+
+        _notificationLoad: function (notificationState, notificationText) {
+            var pjaxResponse = '<div data-name="title"></div>' +
+                    '<div data-name="html"></div>' +
+                    '<ul data-name="notification">' +
+                    '<li data-state="' + notificationState +'">' + notificationText + '</li>' +
+                    '</ul>',
+                uri = 'echo/get/html/?response='+ Y.config.win.encodeURIComponent(pjaxResponse),
+                request = {'params': {'uri': uri}};
+
+            this.service = new Y.eZ.ServerSideViewService({
```

Comment:
this should be done in the `setUp` to separate the initialization from the actual test